### DISCUSSION TITLE
為什麼許多大公司不使用chatGPT？

### DISCUSSION THREAD

**Discussion by WeiYen1103** [1101-GS4719-C/Python-Community-GS4719-C#1644] [Ideas]
我發覺現在的科技公司都標榜人工智慧(AI)技術，但其實這些公司都還沒有找到AI相關產品要如何創造獲利的方法。
首先，生程式AI工具的運作成本很高，我聽系上的教授分享，一些大公司根本不敢把自己公司的資料丟到chatGPT上，請AI幫忙分析，因為我們都不知道這些資料最終會流落到哪去，也不確定會不會有競爭公司在半路就竊聽資料中的機密內容；且準確性也會是一大問題；且畢竟AI不是人類，它生成的文本內容會失去人情味，尤其是在某些最需要人性化色彩的時候。
但這樣相反的若要有生成式AI的幫助，就勢必得自己開發一款專屬自己公司的AI，這樣不僅確保內容不留，也能享有科技帶來的便利，但生程式AI需要有搭載高規個晶片的強大伺服器，還會需要大量的電力，微軟及google近日也在試著研發專屬公司的AI，讓更多使用者都能受益。

**Comment by 1chooo** [comment]
確實很多大公司是不用 ChatGPT 的，因為公司會怕產品的資料流到 ChatGPT 的儲存庫，如果有心人士把相關資訊問出來了！會對公司造成一定的影響。

所以我之前實習的公司，有加入自行開發的工具，同樣是 ChatGPT 回覆，不過只要涉及敏感的關鍵字眼會被工具擋掉，之前 Mentor 直接拿公司合約內容貼給有加入開發工具的 ChatGPT，ChatGPT 便直接無法回，也會提醒該內容可能涉及公司資訊。

**Comment by mrweixd** [comment]
是我也很怕自己公司的資料變成養分被偷走，尤其將機密等級放大成國家等，所以像國科會也會想生出自己的ChatGPT，叫做「TAIDE」。
Source: https://news.pts.org.tw/article/641601

**Comment by Yanxxx7979** [comment]
依我淺薄對chatgpt的使用，以最簡易來說，我們只要登陸了chatgpt ，問的問題都可以保留下來，假如其中有所謂公司的機密資訊，有心之人甚至不需要盜入數據庫等等獲取資訊，只需要盜到個體帳戶便有可能被流出了

**Comment by davidteng00000** [comment]
上一則回覆中 mrweixd 有提到國科會研發的台版ChatGPT「TAIDE」，但是身為它的使用者，我覺得非常難用，訓練資料只有70億筆，相較於GPT4的13000筆有巨大落差。而且只要prompt設的長一點，便會答非所問

**Comment by 710388** [comment]
各個公司還是想要保護自己的隱私吧，不想輕易外洩自己的狀況

**Comment by NcuJerryKao** [comment]
您對於科技公司在AI技術應用上所遇到的挑戰提出了一些有價值的觀點。以下是一些建議回應：
1. 保障數據隱私和安全：對於AI工具所涉及的數據隱私和安全問題，公司應該制定嚴格的數據保護政策，確保用戶數據不會被濫用或外泄，以建立用戶信任。
2. 提高AI準確性和人性化：公司在開發AI工具時應該不斷提升AI的準確性和人性化，透過人機合作、自然語言處理等技術來提供更具人情味和準確性的內容，以提升用戶體驗。
3. 自主開發專屬AI：對於需要保護機密資料或追求個性化需求的公司，自主開發專屬AI可能是一個更好的選擇。通過自主開發AI，公司可以更好地控制資料流向和內容生成過程，確保數據安全和內容品質。
4. 提升AI效能和節能：隨著AI技術的發展，公司可以探索更高效節能的AI解決方案，例如優化算法、硬體設備和能源使用，以降低運作成本和
